setup.py

```diff
@@ -24,7 +24,7 @@
 
 setup_requires = ["Babel>=2.8"]
 
-install_requires = ["invenio-i18n>=1.2.0"]
+install_requires = ["invenio-i18n>=1.2.0", "invenio-admin>=1.2.1,<1.3.0"]
```

Comment:
Is the upper pinning for `invenio-admin` needed here? Usually we manage cross-Invenio-module dependencies, centrally via the `invenio` package dependencies (or in the overlay app), and pin accordingly there.

Reply:
Removed